membership-common/src/main/scala/com/gu/monitoring/SafeLogger.scala

```diff
@@ -37,18 +39,30 @@ class SafeLoggerImpl(logger: Logger) {
     logger.debug(logMessage)
   }
 
-  def info(logMessage: String): Unit = {
+  def infoNoPrefix(logMessage: String): Unit = {
     logger.info(logMessage)
   }
 
-  def warn(logMessage: String): Unit = {
+  def info(logMessage: String)(implicit logPrefix: LogPrefix): Unit = {
+    logger.info(logPrefix.message + ": " + logMessage)
+  }
```

Comment:
this (and others) are the old method with the new parameter required for the prefix.  The new method has the "NoPrefix" suffix to make it clearer you only use them if there's no identity id available.